webapp/packages/data-export-plugin/src/ExportNotification/ExportNotification.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * cloudbeaver - Cloud Database Manager
+ * Copyright (C) 2020 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react';
+import { useEffect, useState } from 'react';
+import { Button as ReakitButton } from 'reakit/Button';
+import styled, { use, css } from 'reshadow';
+
+import { SNACKBAR_STYLES } from '@dbeaver/core/app';
+import { Icon, Button, Loader } from '@dbeaver/core/blocks';
+import { useController } from '@dbeaver/core/di';
+import { NotificationComponentProps } from '@dbeaver/core/eventsLog';
+import { useTranslate } from '@dbeaver/core/localization';
+import { useStyles } from '@dbeaver/core/theming';
+import { EDeferredState } from '@dbeaver/core/utils';
+
+import { Controller } from './Controller';
+
+const styles = css`
+  Loader {
+    margin-right: 16px;
+  }
+  actions {
+    display: flex;
+  }
+  fill {
+    flex: 1;
+  }
+  Button {
+    margin-left: 16px !important;
+  }
+  actions Button {
+    display: flex;
+  }
+  source-name {
+    composes: theme-typography--body2 from global;
+    padding-top: 16px;
+    max-height: 50px;
+    overflow: hidden;
+
+    & pre {
+      margin: 0;
+    }
+  }
+`;
+
+export const ExportNotification = observer(function ExportNotification({
+  notification,
+}: NotificationComponentProps<string>) {
+  const controller = useController(Controller, notification);
+  const translate = useTranslate();
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  return styled(useStyles(SNACKBAR_STYLES, styles))(
+    <notification as="div" {...use({ mounted })} >
+      <notification-header as="div">
+        <Loader loading={controller.isPending} hideMessage />
+        <message as="div">{translate(controller.status)}</message>
+        {!controller.isPending && (
+          <ReakitButton onClick={controller.delete}>
```

Comment:
Create separate companent in @dbeaver/core/blocks
Plugins should know nothing about reakit

Reply:
IconButton implemented